engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActorData.scala

```diff
@@ -22,7 +20,8 @@ final case class WorkflowExecutionDiff(executionStore: Map[JobKey, ExecutionStat
 case class WorkflowExecutionActorData(workflowDescriptor: EngineWorkflowDescriptor,
                                       executionStore: ExecutionStore,
                                       backendJobExecutionActors: Map[JobKey, ActorRef],
-                                      outputStore: OutputStore) extends WdlLookup {
+                                      outputStore: OutputStore,
+                                      restarting: Boolean) extends WdlLookup {
```

Comment:
I'm ok with that, that's what I did in the first place actually. However I changed it because all our actors accept a "Start" and a "Restart" message, so for consistency reasons I wanted to keep that behaviour here. I will make it a props argument but then I will also do it for `WorkflwExecutionActor` and `WorkflowActor` since they are in the same case. An go from `Start`  / `Restart` messages to `Execute` only.